### PR TITLE
Word Counter

### PR DESCRIPTION
**Pull request by KaveenLaksitha**
commit with working project

### CODE REVIEW

**Comment by KaveenLaksitha** [comment]
my task is completed and its ready merge

**Comment by soumik2012** [comment]
Be patient I will check the code then I will merge it

File: Word_Counter/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+function setInput() {
+  var input = document.getElementById("input");
+
+  function isWord(string) {
+    var isAlphaNumeric = false;
```
Comment (soumik2012) [inline Word_Counter/script.js:5]: image
Make the text area wider as I write into it count option is gone? I Can't see the how many words is count fix this.

**Review by soumik2012 — CHANGES_REQUESTED** [review]
A little change is required

File: Word_Counter/script.js (hunk 1 above)
Comment (KaveenLaksitha) [reply Word_Counter/script.js:5]: okay. I'll fix it. Thanks

Comment (KaveenLaksitha) [reply Word_Counter/script.js:5]: @soumik2012  Changes resolved. Please review it. Thanks

**Review by soumik2012 — COMMENTED** [review]
Everything is fine please add Readme.d file  of your project  it should be contained 
- Name of Project
- Demo of Project in .gif how your project is look 
- Description of project 
-  Requirements or Setup instructions
-  Name of Author(s)  
- [Look Here](https://github.com/Tenet-Coding/Hacktoberfest-Projects/blob/master/ConwaysGameOfLife/README.md)

**Review by soumik2012 — APPROVED** [review]
Great @KaveenLaksitha  LGTM 💯

**Comment by KaveenLaksitha** [comment]
@soumik2012  Thank you for your support

**Review by Harshil-Jani — APPROVED** [review]
We are ready to merge :rocket: 
Happy Hacktoberfest :tada:

**Comment by KaveenLaksitha** [comment]
Wish you the same!🎉

**Comment by KaveenLaksitha** [comment]
@Harshil-Jani You can also close issue #14 .

**Pull request merged**